scripts/content-utils/card-tree.ts

```diff
@@ -66,10 +75,18 @@ export function cardsFromTree(tree: CardTree, bindRef?: string): CardData[] {
                             ...triggerRefRemoval,
                         },
                     ),
-                ),
+                ]),
             ],
         ),
-        ...(tree.left ? cardsFromTree(tree.left, leftRef) : []),
-        ...(tree.right ? cardsFromTree(tree.right, rightRef) : []),
+        ...(tree.left && "card" in tree.left
```

Comment:
`"card" in tree.left` is checking if it's a leaf node.

Will we need to check `isCardLeaf(tree.left)` when working with content scripting as well? If so, this could be a named  helper, which also would clarify the real meaning of the prop check on line 81.